samples/client/petstore/php/SwaggerClient-php/phpunit.xml.dist

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="./vendor/autoload.php"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         stopOnFailure="false">
+    <testsuites>
+        <testsuite>
+            <directory>SwaggerClient-php/test/Api</directory>
```

Comment:
Oh, I see the problem, I've tested it with my own config file which sets `srcBasePath: .` so `getPackagePath()` is not required (as `phpunit.xml.dist` gets created **inside** it, not **beside** it). I'll fix it, along with the file you suggested.

Thanks.